src/Services/DomainAuthorization.php

```diff
@@ -80,7 +80,9 @@ protected function createDomainAuthorizationCode(string $callbackLink
      */
     public function createDomainAccessToken(): string
     {
-        $domain = $this->createDomainAuthorizationCode(route('bitaps.payments-forwarding.callback'));
+        //If you are using a local environment, add BITAPS_CALLBACKLINK_FORWARDING to your configuration
+        $callbackLink = env('APP_ENV') === 'local' ? env('BITAPS_CALLBACKLINK_FORWARDING') : route('bitaps.payments-forwarding.callback');
```

Comment:
That should be moved to the config file. We should give an opportunity to change the env variable's names.